processor/elasticapmprocessor/testdata/ecs/elastic_span_db/input.yaml

```diff
@@ -16,6 +16,12 @@ resourceSpans:
         - key: numeric_labels.foo
           value:
             doubleValue: 42.0
+        - key: user.id
+          value:
+            stringValue: user-1234
+        - key: service.framework.name
+          value:
+            stringValue: Node
     scopeSpans:
       - scope: {}
         spans:
```

Comment:
Are there more fields missing in the test here? Shouldn't there be many more changes in the test output file, looking at the `ecs_translation` changes?